docs/02.md

```diff
@@ -0,0 +1,143 @@
+for all the workflows packit covers.
+
+Let's get through what's new:
+
+1. We have decided to rename two keys in our config file so they are more
+   descriptive. Old names still work but they are deprecated:
+    * `package_name` → `downstream_package_name`
+    * `upstream_name` → `upstream_project_name`
+2. You don't need to touch dist-git at all when getting your new upstream
+   release into Fedora, you can stay in your upstream repository and just fire
+   off a bunch of packit calls:
+    * `packit propose-update` to create a pull request in Fedora dist-git
+    * `packit build` to build once it's merged
+    * and finally, `packit create-update` creates a new bodhi update (if you
+      chose a stable Fedora release)
+3. Packit now has a `srpm` command which creates an SRPM out of the local
+   content of your upstream repository.
+4. You can now use packit to sync files from your dist-git repo back into
+   upstream (mainly to keep spec files in sync). `sync-from-downstream` is the
+   command.
+5. Command `propose-update` received numerous improvements:
+    * You can pick upstream version to use.
+    * Packit will NOT check out the git ref with the upstream release if you
+      specify `--local-content`
+    * It's possible to force packit to execute `fedpkg new-sources` using
+      `--force-new-sources` and bypass the caching mechanism.
+
+
+## Installation
+
+Please make sure you are installing `0.2.0`:
+```
+$ dnf install --enablerepo=updates-testing packit
+```
+
+Or
+
+```
+$ pip3 install --user packitos
+```
+
+You can also install packit from master branch, if you are brave enough:
+
+```
+$ pip3 install --user git+https://github.com/packit-service/packit
+```
+
+
+## How are we using packit?
+
+I'd like to show you how we used packit to bring a new upstream
+release of ogr into Fedora, a library which packit is using.
+
+Once we have performed an upstream release of ogr, we can propose an update in
+dist-git:
+```
+$ git clone https://github.com/packit-service/ogr && cd ogr/
+
+$ packit propose-update
+INFO: Running 'anitya' versioneer
+Version in upstream registries is '0.0.3'.
+Version in spec file is '0.0.3'.
+Picking version of the latest release from the upstream registry over spec file.
+Checking out upstream version 0.0.3
+Using 'master' dist-git branch
+Cloning repo: https://src.fedoraproject.org/rpms/python-ogr.git -> /tmp/tmpb9xlvdhj
+Syncing /home/tt/g/user-cont/ogr/python-ogr.spec
+Archive ogr-0.0.3.tar.gz found in lookaside cache (skipping upload).
+ERROR    Cmd('git') failed due to: exit code(1)
+  cmdline: git commit -s -m [packit] 0.0.3 upstream release -m Upstream tag: 0.0.3
+Upstream commit: 059d21080a7849acff4626b6e0ec61830d537ac4
+
+  stdout: 'On branch 0.0.3-master-update
+nothing to commit, working tree clean'
+```
+
+Whoops, it seems that I have messed up, I forgot to bump the spec file in the
+upstream repo when doing the release. I will bump it locally and utilize
+`--local-content` argument:
```

Comment:
What about the example of reverse-sync of the `specfile`?

Reply:
hm?